adding_data_brief.qmd

```diff
@@ -23,6 +23,30 @@ schema <- get_schema()
 definitions <- austraits$definitions
 ```
 
-This chapter gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
+This section gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
 
-XXX add details
+It is important that all steps are followed so that our automated workflow proceeds without problems.
+
+1. Clone the `traits.build` repository from github
+2. Create a new branch in the repo, named for the new `dataset_id` in `author_year` format, e.g. `Gallagher_2014`.
+3. Create a new folder within the folder `data` with the name `dataset_id`, e.g. `Gallagher_2014`.
+4. Prepare the file `data.csv` and place it within the new folder ([details](#csv_file) here).
+5. Prepare the file `metadata.yml` and place it within the new folder ([details](#metadata_file) here).
+6. Add the new study into the build framework and rebuild AusTraits, by running `build_setup_pipeline()`.
+
+This step updates the file `remake.yml` with appropriate rules for the new dataset; similarly if you remove datasets, do the same. (At this stage, [remake](https://github.com/richfitz/remake) offers no looping constructs, so for now we generate the remake file using: [whisker](https://github.com/edwindj/whisker).)
```

Comment:
There's now 3 build options, base R, furrr, remake. I suggest new users use base. Add link to section on different pipelines